<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_1">
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Hotkeys</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="hotkeyCtrl">
        <property name="icon">
         <iconset resource="UMWP_Autochanger.qrc">
          <normaloff>:/key/ctrl</normaloff>:/key/ctrl</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="hotkeyAlt">
        <property name="icon">
         <iconset resource="UMWP_Autochanger.qrc">
          <normaloff>:/key/alt</normaloff>:/key/alt</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="hotkeyShift">
        <property name="icon">
         <iconset resource="UMWP_Autochanger.qrc">
          <normaloff>:/key/shift</normaloff>:/key/shift</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="hotkeyWin">
        <property name="icon">
         <iconset resource="UMWP_Autochanger.qrc">
          <normaloff>:/key/win</normaloff>:/key/win</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="3">
    <widget class="QGroupBox" name="groupBox_1">
     <property name="title">
      <string>Configuration</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_1">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Delay (seconds) :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="optionDelay">
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>3600</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="optionMinimize">
        <property name="text">
         <string>Minimize on startup</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="optionCheckFiles">
        <property name="text">
         <string>Check files periodically</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="optionCheckUpdates">
        <property name="text">
         <string>Check updates</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="optionAutostart">
        <property name="text">
         <string>Start with Windows</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="UMWP_Autochanger.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hotkeyAlt</sender>
   <signal>toggled(bool)</signal>
   <receiver>ConfigDialog</receiver>
   <slot>hotkeysChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hotkeyCtrl</sender>
   <signal>toggled(bool)</signal>
   <receiver>ConfigDialog</receiver>
   <slot>hotkeysChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hotkeyShift</sender>
   <signal>toggled(bool)</signal>
   <receiver>ConfigDialog</receiver>
   <slot>hotkeysChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hotkeyWin</sender>
   <signal>toggled(bool)</signal>
   <receiver>ConfigDialog</receiver>
   <slot>hotkeysChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>hotkeysChanged()</slot>
 </slots>
</ui>
